release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -621,6 +621,90 @@ For more information, see xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-
 [id="ocp-4-11-monitoring"]
 === Monitoring
 
+The monitoring stack for this release includes the following new and modified features.
+
+[id=ocp-4-11-monitoring-updates-to-monitoring-stack-components-and-dependencies]
+==== Updates to monitoring stack components and dependencies
+
+Updates to versions of monitoring stack components and dependencies include the following:
+
+* Alertmanager to 0.24.0
+* kube-state-metrics to 2.5.0
+* Prometheus to 2.36.2
+* Prometheus operator to 0.57.0
+* Thanos to 0.26.0
+
+[id=ocp-4-11-monitoring-changes-to-alerting-rules]
+==== Changes to alerting rules
+
+[NOTE]
+====
+Red Hat does not guarantee backward compatibility for recording rules or alerting rules.
```

Comment:
By `backward compatibility` I take that to mean that Red Hat might not be able to record or alert rules for older versions, am I right? If `backward compatibility` is a phrase users would know, then keep it. Otherwise, consider rephrasing to something like:

`Red Hat does not guarantee that it can record or alert rules for older versions.` 

or 

`Red Hat does not guarantee that it can record or alert rules for versions 4.10 or older.`